static/js/main.js

```diff
@@ -375,6 +396,13 @@ function takeScreenshot(source, x, y, width, height, format = 'image/jpeg') {
     return canvas.toDataURL(format);
 }
 
+function deleteScreenshot(screenshot_id) {
+    // Find screenshot area to remove.
+    document.querySelector(`.screenshot[data-id="${screenshot_id}"]`).remove();
+    
```

Comment:
I would probably remove this whitespace and let this visual block conceptually be be "removing the screenshot".
You can keep it as two visual blocks ("removing from the dom" and "removing from the data").
Either is fine.

Note that I am only making this comment since there are other comments on this pass.
This comment would not be something that is worthy of another PR pass by itself.